includes/polling/os/allied-wireless.inc.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+/*Wireless support for
+AT-TQ2403, AT-TQ2403EX, AT-TQ2450, AT-TQ3600,
+AT-TQ3200, AT-TQ3400, AT-TQ4400, AT-TQ4600, AT-TQ4400e
+
+Use sysDescr to get Hardware, SW version*/
+
+list($a,$b,$c) = explode(' ', $device['sysDescr']);
+    $hardware = $a;
```

Comment:
You could inline $hardware and $version into the list() here to avoid temporary variables.

Reply:
Nice. Didn't think of that.